src/ucp/core/ucp_listener.c

```diff
@@ -172,6 +173,26 @@ static void ucp_listener_conn_request_callback(uct_iface_h tl_iface, void *arg,
     ucp_worker_signal_internal(listener->worker);
 }
 
+ucs_status_t ucp_conn_request_query(ucp_conn_request_h conn_request,
+                                    ucp_conn_request_attr_t *attr)
+{
+    ucs_status_t status;
+
+    if (attr->field_mask & UCP_CONN_REQUEST_ATTR_FIELD_CLIENT_ADDR) {
+        if (conn_request->client_address.ss_family == 0) {
+            return UCS_ERR_UNSUPPORTED;
```

Comment:
how could it be 0?

Reply:
in the previous flow (without cm) - https://github.com/openucx/ucx/pull/4867/files#diff-556db94b24d5e7dee4a5f491a9877b8fR163

Reply:
yes, right, i've almost forgot we have that..